tests/test_json_validation.py

```diff
@@ -0,0 +1,42 @@
+import os
+import json
+import jsonschema
+from jsonschema import validate
```

Comment:
Same point about imports here. I'd recommend adding `SchemaError` and `ValidationError` to the second line and changing the try catch blocks later in the module.